README.md

```diff
@@ -48,6 +48,27 @@ Snapshotting configuration; setting values in this list will save the database t
 
 Database compression and location configuration.
 
+    redis_maxmemory: 0
+    
+Limit memory usage to the specified amount of bytes. When the memory limit is reached Redis will try to remove keys according to the eviction policy selected (see redis_maxmemory_policy).
+Leave at 0 for no limit.
+
+    redis_maxmemory_policy: "allkeys-lru"
+    
+How Redis will select what to remove when maxmemory is reached. You can select among five behaviors:
```

Comment:
For this comment, is there some official docs page you can link to, rather than listing all the options below? For something that most users would probably not be changing, I'd rather link to a docs page explaining things in depth than have to maintain the documentation as part of this role.